app/helpers/template_paths_helper.rb

```diff
@@ -0,0 +1,14 @@
+module TemplatePathsHelper
+  def template_path_for(type, route = 'path')
```

Comment:
If this method should act as extension point, that's another reason for me to have the method for just returning the prefix, that would be overriden from the plugin if needed, and the other methods for generating the _path_for (or the url_for if needed).